blog/2023-06-28-service-description-filters.md

```diff
@@ -4,8 +4,8 @@ authorURL: https://github.com/emontnemery
 title: "Service call description filters"
 ---
 
-Service call fields now support filtering, to avoid showing service call parameters that are not supported by an entity to the user. For example, the `brightness` service call parameter for `light.turn_on` will only be shown if the light supports brightness.
-
-This is an extension of the previously supported possibility to filter out entities that don't support a certain service call.
+Service call descriptions now support filtering, to avoid showing entieis not supported by a service calls and service call parameters that are not supported by an entity to the user. For example:
```

Comment:
entities typo. also probably not necessary to mention 'now' anymore.

Reply:
Good call, typo fixed + some rewording